vars/deploy-on-openstack.yml

```diff
@@ -18,10 +18,7 @@ deploy_on_openstack_sesnode_userdata: |
 deploy_on_openstack_repos_to_configure:
   SLE12SP3-product: http://provo-clouddata.cloud.suse.de/repos/x86_64/SLES12-SP3-Pool/
   SLE12SP3-update: http://provo-clouddata.cloud.suse.de/repos/x86_64/SLES12-SP3-Updates/
-  SLE12SP3SDK-product: http://provo-clouddata.cloud.suse.de/repos/x86_64/SLE12-SP3-SDK-Pool/
-  SLE12SP3SDK-update: http://provo-clouddata.cloud.suse.de/repos/x86_64/SLE12-SP3-SDK-Updates/
   SUSE-CA: http://download.suse.de/ibs/SUSE:/CA/SLE_12_SP3/
-  OpenStack-Cloud8-product: http://provo-clouddata.cloud.suse.de/repos/x86_64/SUSE-OpenStack-Cloud-8-Pool/
```

Comment:
We can't delete these 3. They're still referenced by the ses playbooks. See `deploy_on_openstack_ses_repos_per_imagename` a few lines deeper.

The SES playbook (https://github.com/SUSE-Cloud/socok8s/blob/master/playbooks/openstack-ses_aio_hostconfig.yml#L5) installs a couple of packages from some of those repos. I am sure most of that is unneeded, but that should be cleaned up in a separate PR IMO.

Reply:
@rhafer this could be removed after https://github.com/SUSE-Cloud/socok8s/pull/296 which I was hoping would be merged by now

Reply:
Ok. I didn't see that one.